src/index.html

```diff
@@ -7,6 +7,402 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header" id="home">
+      <div class="header__logo">
+        <a href="./index.html" class="header__logo-link">
+          <img src="./images/logo.png" alt="MOYO logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__links is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__links"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__links" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div class="container__wrapper-cards">
+        <div class="container__cards card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <div class="card__product">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
+          <div class="card__review">
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
```

Comment:
```suggestion
              <div class="stars__star"></div>
```
Why do you need block `star`? You don't even use it
It's just an element of block `stars`